### PR TITLE
Diaper request link to pickup sheet

### PR DESCRIPTION
**Pull request by mlpinit**
- Add link to pickup sheets on the partner request page.
- Add misc child item requests relationship to partner requests

Screen Shot 2019-10-22 at 5 08 24 PM

### CODE REVIEW

File: app/models/partner_request.rb [hunk 1]
```diff
@@ -15,11 +15,14 @@ class PartnerRequest < ApplicationRecord
   belongs_to :partner
 
   has_many :item_requests, dependent: :destroy
-  accepts_nested_attributes_for :item_requests, allow_destroy: true, reject_if: proc { |attributes| attributes["quantity"].blank? }
+  accepts_nested_attributes_for :item_requests, allow_destroy: true,
```
Comment (benreyn) [inline app/models/partner_request.rb]: ❤️ This reads so much better split up on two lines. 🙌

File: app/views/partner_requests/index.html.erb [hunk 2]
```diff
@@ -32,6 +35,13 @@
           <tr>
             <th scope="row"><%= link_to partner_request.id, partner_request_path(partner_request) %></th>
             <td><%= partner_request.item_requests.map {|item| "#{item.quantity} of #{item.name}"}.join(", ") %></td>
+            <% if Flipper[:family_requests].enabled?(@partner) %>
+              <td>
+                <% if partner_request.for_families? %>
+                  <%= link_to "Pickup Sheet", family_request_pickup_sheets_path(partner_request) %>
```
Comment (benreyn) [inline app/views/partner_requests/index.html.erb]: Probably don't need to address this now, but I wonder if we should eventually make this a bootstrap button with a meaningful icon?

File: spec/features/family_requests_feature_spec.rb [hunk 3]
```diff
@@ -36,4 +36,23 @@
       expect(PartnerRequest.last.for_families?).to eq(true)
     end
   end
+
+  describe "pickup sheet link" do
+    scenario "it displays a link to a pickup sheet" do
+      stub_successful_items_partner_request
```
Comment (benreyn) [inline spec/features/family_requests_feature_spec.rb]: These methods are really pulling their weight. 😉

**Comment by benreyn** [comment]
@mlpinit, I think you got got by rubocop. Once thats fixed lets merge this baby!

File: app/views/partner_requests/index.html.erb (hunk 2 above)
Comment (mlpinit) [reply app/views/partner_requests/index.html.erb]: Happy to do that now. I'll use the same icon as on the pickup sheets page.

Comment (mlpinit) [reply app/views/partner_requests/index.html.erb]: Resolved with: 976415f

**Pull request merged**